test/unit/rubygem_searchable_test.rb

```diff
@@ -41,8 +41,8 @@ class RubygemSearchableTest < ActiveSupport::TestCase
 
   context 'filter' do
     setup do
-      example_1 = create(:rubygem, name: "example_1")
-      example_2 = create(:rubygem, name: "example_2")
```

Comment:
Changing the name was the easiest workaround, as the test is for `filter` over yanked gems and name of gem doesn't matter here.